src/components/Note/Note.tsx

```diff
@@ -0,0 +1,49 @@
+
+import React from 'react';
```

Comment:
The formatting for this file is a bit wonky. You might want to look into something like Prettier to automatically clean up the code when you save it.